### PR TITLE
show breakpoints in source code, also remove some bold

### PR DESCRIPTION
**Pull request by KristofferC**
Excessive use of bold is the bane of my existence.

deb

### CODE REVIEW

File: src/printing.jl [hunk 1]
```diff
@@ -225,15 +238,19 @@ function print_sourcecode(io::IO, code::String, line::Integer, defline::Integer)
...
-                        "" :
-                        " "
+        break_on_line = lineno in breakpoint_lines
+        prefix = "", :normal
+        break_on_line           && (prefix = ('●', :light_red))
+        lineno == current_line  && (prefix = ('>', :yellow))
         printstyled(io,
-            string(lpad(string(prefix, lineno), stoplinelength , " "), "  "),
-            color = lineno == current_line ? :yellow : :bold)
+            string(lpad(string(prefix[1], lineno), stoplinelength , " "), "  "),
```
Comment (pfitzseb) [inline src/printing.jl]: Maybe
```
string(string(prefix[1], lpad(lineno, stoplinelength, " "), "  "),
```
instead? That way the breakpoints/current line indicators are consistently in one column (much like in an editor).

Comment (KristofferC) [reply src/printing.jl]: I thought about that too but not sure it looked better:

deb2

Comment (pfitzseb) [reply src/printing.jl]: I like it better, fwiw ;)
Would also add a space between the symbols and the line numbers, but maybe people feel like their horizontal space is too precious for that?

Comment (KristofferC) [reply src/printing.jl]: Updated to your suggestion :)

**Comment by KristofferC** [comment]
Perhaps we should have a different color for conditional breakpoints?

**Comment by pfitzseb** [comment]
IIRC Juno uses yellow for conditional breakpoints, and full vs empty circles for enabled/disabled breakpoints. 

Would probably be good to be consistent (I'm not attached to the above suggestion at all though).

**Comment by KristofferC** [comment]
I'm thinking about `◐` for conditional and `○` for disabled. Thoughts?

**Comment by pfitzseb** [comment]
I like it. Distinguishing conditional and non-conditional disabled breakpoints doesn't seem super important.

**Comment by KristofferC** [comment]
Agreed

**Comment by timholy** [comment]
Whatever we do we should make it the same throughout (see `JuliaInterpreter.breakpointchar`).

**Comment by KristofferC** [comment]
Maybe, I don't think it is too bad for the frontend to chose how they display things (that is pretty much the only point of a frontend heh). I do see the reason of having things consistent but people who look at the output for a framecodes and those that look at output of Debugger.jl are likely quite different.

**Comment by timholy** [comment]
Good point! I'm fine with it.

**Pull request merged**